HousingSearchApi.Tests/HousingSearchApi.Tests.csproj

```diff
@@ -49,7 +49,6 @@
     <PackageReference Include="Docker.DotNet" Version="3.125.4" />
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
     <PackageReference Include="Hackney.Core.ElasticSearch" Version="1.45.0" />
-    <PackageReference Include="Hackney.Shared.HousingSearch" Version="0.6.0" />
```

Comment:
no tests need this package?

Reply:
test management will be executed in the Unit Testing PR - because of following the small PR the unit testing would be implemented in another pr